.examples/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: '3.9'
```

Comment:
@rib-reply can you please review?

Reply:
If we don't want to touch the original files, I think it might be the best to separate these changes into a new file, so you can hold off the review until next week

Reply:
Indeed that is one of the problems. I must repeat, I can not verify this for Docker For Desktop/docker-compose on MacOS (since I am forced to use windows, because of my project). Your docker would publish your port on all network interfaces as 0.0.0.0 - if you did not configure the standard IP address in your settings as 127.0.0.1 (loopback adapter) In case this docker-compose file is only relevant for local development (and each service is so). To prevent this you can either set the port mapping from container to host like:

- port: 127.0.0.1:80:80

Or setup your docker daemon to use this by standard. To do this on your mac using Docker For Desktop:
go to Docker For Desktop -> preferences > daemon > advanced in the toolbar and add the key / value to the json overlay:

{
	"ip" : "127.0.0.1",
	...
}

Verify this by using another machine and use nmap -Pn {ip of the host}

Care that we rely on our developers to setup their docker for desktop daemon correctly. Personally I would definitely not count on that, if I would be accountable.

cheers, Igor

Reply:
same goes for the problem in the README

docker run -p 8080:8080 urlaubsverwaltung/urlaubsverwaltung:${version}

would publish the containers port 8080 on the hosts port 8080 for all network interfaces